ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -847,3 +847,75 @@ def test_torch_tensorsolve(
         A=A,
         B=B,
     )
+
+
+# lu_factor
+@st.composite
+def _lu_factor_helper(draw):
+    # generate input matrix of shape (*, m, n) and where '*' is one or more batch dimensions
+    input_dtype = draw(
+        helpers.get_dtypes("float", index=1, full=True)
+    )
+
+    dim1 = draw(helpers.ints(min_value=2, max_value=5))
+    dim2 = draw(helpers.ints(min_value=2, max_value=5))
+    batch_dim = draw(helpers.get_shape(min_dim_size=0, max_dim_size=5))
+
+    if batch_dim == 0:
+        input_matrix = draw(
+            helpers.array_values(
+                dtype=input_dtype,
```

Comment:
The problem was caused by this line, you are passing a list to `dtype` while it should take a string; `'float'` for example, and since the `dtype` is shared, then we can just use the first element of the `dtype` list. `dtype=input_dtype[0]` instead of `dtype=input_dtype` should work